src/seedu/addressbook/AddressBook.java

```diff
@@ -365,26 +384,28 @@ private static void loadDataFromStorage() {
      * @return  feedback about how the command was executed
      */
     private static String executeCommand(String userInputString) {
-        final String[] commandTypeAndParams = splitCommandWordAndArgs(userInputString);
-        final String commandType = commandTypeAndParams[0];
-        final String commandArgs = commandTypeAndParams[1];
-        switch (commandType) {
+        final String[] COMMAND_TYPE_AND_PARAMS = splitCommandWordAndArgs(userInputString);
+        final String COMMAND_TYPE = COMMAND_TYPE_AND_PARAMS[0];
+        final String COMMAND_ARGS = COMMAND_TYPE_AND_PARAMS[1];
+        switch (COMMAND_TYPE) {
```

Comment:
Note that this is not the right use of constant variables and therefore the variables should not be in all caps.